File: applied_crypto copy/playground/lesson1/bf_caesar_cipher.rb
require 'pry'
require_relative './ruby_resources/historic_ciphers'


phrase = "hvwgw gqozz srobv wghcf wqozq wdvsf psqoi gswhw gpfcy sbhbl"
bf_result = HistoricCiphers::CaesarCipher.brute_force_decrypt(phrase)

# puts bf_result

# 14=>"THISI SCALL EDANH ISTOR ICALC IPHER BECAU SEITI SBROK ENTNX",
# THIS IS CALLEC AN HISTORICAL CIPHER BECAUSE IT IS BROKEN TNX

phrase2 = "sahyk iapkp daben opzwu kbpda naopk bukqn hebau"

bf_result2 = HistoricCiphers::CaesarCipher.brute_force_decrypt(phrase2)

puts bf_result2

# 22=>"WELCO METOT HEFIR STDAY OFTHE RESTO FYOUR LIFEY",
# WELCOME TO THE FIRST DAY OF THE REST OF YOUR LIFE Y
binding.pry

File: applied_crypto copy/playground/lesson1/bf_caesar_cipher.py
from python_resources.ciphers import bf_decrypt_cc

msg1 = "hvwgw gqozz srobv wghcf wqozq wdvsf psqoi gswhw gpfcy sbhbl"
msg2 = "sahyk iapkp daben opzwu kbpda naopk bukqn hebau"
bf_res1 = bf_decrypt_cc(msg1)
bf_res2 = bf_decrypt_cc(msg2)

print(bf_res1) # (14, 'THISISCALLEDANHISTORICALCIPHERBECAUSEITISBROKENTNX')
print("---------------------------------------")
print(bf_res2) #  (22, 'WELCOMETOTHEFIRSTDAYOFTHERESTOFYOURLIFEY')